static/lang/re-en.json

```diff
@@ -589,6 +589,12 @@
             "Falrok": {
                 "LazuriteTerrainSwitch": "In lazurite infused terrain"
             },
+            "Ghaele" : {
+                "ChooseWeakness": {
+                    "Damage": "Choose Weakness - Damage",
+                    "Resistance": "Choose Weakness - Resistance"
+                }
+            },
```

Comment:
I think these can be made generic enough to put in with the other generic prompts.

Reply:
And make them well-formed sentences like all the others. What is "Choose Weakness - Resistance"?

Reply:
Good feedback.

Any suggestions? I was hoping to tie it to the ability, which is "Choose Weakness", but I agree that it makes the name confusing.

Perhaps just "Elemental Damage" and "Elemental Resistance"? It's generic so it can but used in other places, but I'm concerned that there's no hint to tie it to the ability.

[Ghaele](https://2e.aonprd.com/Monsters.aspx?ID=37)

Reply:
"Select a Damage Type" or "Select a Weakness" or something along those lines.